packages/aws-cdk-lib/aws-autoscaling/lib/step-scaling-action.ts

```diff
@@ -71,10 +72,16 @@ export class StepScalingAction extends Construct {
   constructor(scope: Construct, id: string, props: StepScalingActionProps) {
     super(scope, id);
 
+    // Specify cooldown property in StepScaling policy type is ineffective and may cause deployment failure
+    // in certain regions. We can't simply remove the property since it break existing users. Since setting
+    // this value is ineffective, we can safely ignore the value of this property with a warning.
+    if (props.cooldown && props.cooldown.toSeconds().toString()) {
```

Comment:
Do we have to check for `props.cooldown.toSeconds().toString()` here? Should `props.cooldown` be enough?